0001-kaa-protocol/README.md

```diff
@@ -93,40 +106,88 @@ Examples of extension specific paths are:
 /json/status
 ```
 
-### Endpoint-specific and nonspecific extensions
+### Payload format specifier
+Extensions may support multiple payload formats.
+In that case, it is wise to add payload format specifier.
 
-There are two large classes of extensions: those that are endpoint-specific and require an endpoint to operate on, and those that do not. While most extensions are endpoint-specific (data collection, profiling, configuration, etc.), there are extensions that are not (endpoint registration).
+For example, use `/json` for JSON-formatted payload, and `/protobuf/<scheme_id>` for protocol buffers.
```

Comment:
Can we define json is:

1. Required to be supported on server (client may fall-back to json in any case)
2. The default format (and thus can be omited in the resource path)
?